### ISSUE TITLE
error: no-assigning-return-values: Do not assign return value of Cy command

### ISSUE THREAD

**Issue by jennifer-shehane** [cypress-io/eslint-plugin-cypress#12]
I just added the `eslint-plugin-cypress` to the Cypress documentation. So now our linter is running against our examples in our documentation. 

There are some examples we have in the docs that are erroring in the eslint-plugin, which I have to think should not be erroring since they are recommended uses of cypress. Can we make this rule exclude assignments of `cy.spy` or `cy.stub`?

1. `error  cypress/no-assigning-return-values: Do not assign the return value of a Cypress command` There are other examples in stub as well like `const stub = cy.stub()`
  https://docs.cypress.io/api/commands/spy.html#Aliases
  ```js
  const obj = {
    foo () {}
  }
  const spy = cy.spy(obj, 'foo').as('anyArgs')
  const withFoo = spy.withArgs('foo').as('withFoo')
  
  obj.foo()
  expect(spy).to.be.called
  expect(withFoo).to.be.called // purposefully failing assertion
  ```

**Comment by chrisbreiding** [comment]
Good catch. We'll have to whitelist those commands so they don't get caught by that rule.

**Comment by chrisbreiding** [comment]
Released in [2.1.3](https://github.com/cypress-io/eslint-plugin-cypress/releases/tag/v2.1.3)

**Comment by benface** [comment]
@chrisbreiding Could there have been a regression since v2.1.3? I'm seeing this in v2.10.3:

```
error: Do not assign the return value of a Cypress command (cypress/no-assigning-return-values) at cypress/integration/examples/spies_stubs_clocks.spec.js:12:5:
  10 |     }
  11 | 
> 12 |     const spy = cy.spy(obj, 'foo').as('anyArgs')
     |     ^
  13 | 
  14 |     obj.foo()
  15 | 
```

```
error: Do not assign the return value of a Cypress command (cypress/no-assigning-return-values) at cypress/integration/examples/spies_stubs_clocks.spec.js:60:5:
  58 |     }
  59 | 
> 60 |     const stub = cy.stub(obj, 'foo').as('foo')
     |     ^
  61 | 
  62 |     obj.foo('foo', 'bar')
  63 | 
```

Using ESLint 6.8.0 with the following config:
```js
{
  extends: [
    'plugin:vue/recommended',
    'standard',
    'plugin:prettier/recommended',
    'prettier/vue',
    'prettier/standard',
    'plugin:cypress/recommended',
    'plugin:chai-friendly/recommended',
  ],
}
```
